libdnf/hy-goal.c

```diff
@@ -46,6 +46,7 @@
 #include "hy-selector-private.h"
 #include "hy-util.h"
 
+#define BLOCK_SIZE 15
```

Comment:
It seems to be optimal value for the purpose. Similarly to
libdnf/dnf-sack.c:    const int BLOCK_SIZE = 31;
libdnf/hy-package.c:#define BLOCK_SIZE 31
libdnf/hy-query.c:#define BLOCK_SIZE 15
libdnf/hy-subject.c:    const int BLOCK_SIZE = 6;
python/hawkey/subject-py.c:    const int BLOCK_SIZE = 6;